app/vlstorage/main.go

```diff
@@ -370,6 +381,10 @@ func writeStorageMetrics(w io.Writer, strg *logstorage.Storage) {
 	metrics.WriteCounterUint64(w, `vl_merges_total{type="storage/small"}`, ss.SmallPartMergesTotal)
 	metrics.WriteCounterUint64(w, `vl_merges_total{type="storage/big"}`, ss.BigPartMergesTotal)
 
+	metrics.WriteCounterUint64(w, `vl_rows_merge_total{type="storage/inmemory"}`, ss.InmemoryMergeRowsTotal)
```

Comment:
It is better to use consistent naming with VictoriaMetrics for these metrics: `vl_rows_merged_total` - see https://github.com/VictoriaMetrics/VictoriaMetrics/blob/deec361a64be416ec9e027adeabe0efceb70036c/app/vmstorage/main.go#L477-L479 . It would be great also to use the consistent naming of the `logstorage.StorageStats` fields for these metrics, e.g. `InmemoryRowsMerged` for this particular metrics.